Rear Rider/Rear Rider/ConfigurationOptions/OptionsView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  OptionsView.swift
+//  Rear Rider
+//
+//  Created by Bobby Palko on 10/25/22.
+//
+
+import SwiftUI
+
+// This binding extension lets us attach our change handler directly to the binding instead of the view, letting us place the observer next to the thing its observing instead of having onChange() modifiers everywhere.
+extension Binding {
+    func onChange(_ handler: @escaping (Value) -> Void) -> Binding<Value> {
+        Binding(
+            get: { self.wrappedValue },
+            set: { newValue in
+                self.wrappedValue = newValue
+                handler(newValue)
+            }
+        )
+    }
+}
+
+struct OptionsView: View {
+    @ObservedObject var conf = UserConfig()
```

Comment:
This should be initialized in Rear_RiderApp.swift as an StateObject similar to the BLEManager because we want the settings to be loaded as soon as the app is started, especially the sound file.